pkg/cp/cmp/cmp.go

```diff
@@ -37,6 +37,9 @@ func isEqualFile(fpath1, fpath2 string) error {
 	return nil
 }
 
+// For testing purpose
+var readDirName = readDirNames
```

Comment:
See this for a way to do testing for readdir, 
https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/io/fs/readdir_test.go
it's a pretty good model of how to implement testing for file systems and files.